src/System.Private.ServiceModel/src/System/ServiceModel/Security/SecurityUtils.cs

```diff
@@ -776,6 +776,7 @@ internal static NetworkCredential GetNetworkCredentialOrDefault(NetworkCredentia
 
         internal static class NetworkCredentialHelper
         {
+            static string currentUser = string.Empty;
```

Comment:
>currentUser [](start = 26, length = 11)

We should call this s_currentUser and declare it private to be consistent with recommended code formatting conventions.